contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -337,6 +339,31 @@ namespace eosiosystem {
 
    typedef eosio::multi_index< "rexpool"_n, rex_pool > rex_pool_table;
 
+   // `rex_return_pool` structure underlying the rex return pool table. A rex return pool table entry is defined by:
+   // - `version` defaulted to zero,
+   // - `last_update_time` the last time returns from renting, ram fees, and name bids were added to the rex pool,
+   // - `current_rate_of_increase` current amount to be added to the rex pool at a rate of per 30 days,
+   // - `cummulative_proceeds` bookkeeping variable that tracks fees added to rex pool up to current time,
+   // - `proceeds` bookkeeping variable that tracks fees added to rex return pool up to current time,
+   // - `return_buckets` 12-hour buckets containing amounts to be added to the rex pool and the times they become effective 
+   struct [[eosio::table,eosio::contract("eosio.system")]] rex_return_pool {
+      uint8_t                       version = 0;
+      
+      time_point                    last_update_time;
+      int64_t                       current_rate_of_increase = 0;
+      int64_t                       cummulative_proceeds = 0;
+      int64_t                       proceeds = 0;
```

Comment:
I don't think `cummulative_proceeds` and `proceeds` will be necessary anymore when you track the proceeds and distribution rate at a per bucket level.